packages/usa-input-mask/src/styles/_usa-input-mask.scss

```diff
@@ -0,0 +1,28 @@
+@use "uswds-core" as *;
+
+.usa-input-mask {
+  position: relative;
+  line-height: 1;
+
+  &--content {
+    position: absolute;
+    left: 2px;
+    top: 0;
+    pointer-events: none;
+    color: color("gray-50");
+    font-family: family("sans");
```

Comment:
Can this be defined with $theme-form-font-family? (Or declare the font-family using the placeholder below)

Reply:
Done